src/connectors/exchangeRate/index.ts

```diff
@@ -151,31 +131,6 @@ export class ExchangeRate {
     return pairs
   }
 
-  private fetchTokenRates = async (): Promise<Rate[]> => {
-    const data = await this.requestCoingeckoAPI(
-      tokenCurrencies,
-      quoteCurrencies
-    )
-    const rates: Rate[] = []
-    for (const t of tokenCurrencies) {
-      for (const q of quoteCurrencies) {
-        rates.push(this.parseCoingeckoData(data, { from: t, to: q }))
-      }
-    }
-    return rates
-  }
-
-  private fetchFiatRates = async (): Promise<Rate[]> => {
-    const rates: Rate[] = []
-    for (const f of fiatCurrencies) {
-      const data = await this.requestExchangeRatesDataAPI(f, quoteCurrencies)
-      for (const q of quoteCurrencies) {
-        rates.push(this.parseExchangeRateData(data, { from: f, to: q }))
-      }
-    }
-    return rates
-  }
-
   private parseCoingeckoData = (
```

Comment:
shall we still keep these methods?

Reply:
In practice, we will not query apis  directly to get exchange rates  ,  as  exchange rates have store in cache for 10 days long. But considering the situation we setup new env  for matters server ( matters-server-stage, for example),  exchange rates will not exist in cache until cron job run its first time.